stdlib/src/collections/list.mojo

```diff
@@ -418,6 +419,25 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
             earlier_idx += 1
             later_idx -= 1
 
+    # TODO: Modify this to be regular method when issue 1876 is resolved
+    @staticmethod
+    fn index[C: ComparableCollectionElement](self: List[C], owned value: C, start: Int = 0, end: Optional[Int] = None) raises -> Int:
+        """Returns the index of the first occurrence of a value in a list; raises Error if not found."""
+        var normalized_start = (self.size + start) if start < 0 else start
+        var normalized_end = self.size if end is None else (self.size + end if end < 0 else (self.size if end > self.size else end))
```

Comment:
That's a lot of logic in the same line, and can make it quite hard to read. Can you split it into multiple lines?

Reply:
Done.